Sources/BraintreePaymentFlow/Public/BraintreePaymentFlow/BTLocalPaymentRequest.h

```diff
@@ -22,6 +22,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable, copy) NSString *paymentType;
 
+/**
+ The country code of the local payment.
+
+ This value must be one of the supported country codes for a given local payment type listed at the link below. For local payments supported in multiple countries, this value may determine which banks are presented to the customer.
+
+ https://developers.braintreepayments.com/guides/local-payment-methods/client-side-custom/ios/v4#invoke-payment-flow
+ */
+@property (nonatomic, nullable, copy) NSString *paymentTypeCountryCode;
```

Comment:
Would it be more clear to just call it `countryCode`, since that is what the value is? `paymentTypeCountryCode` is throwing me off a little.

Reply:
> For local payments supported in multiple countries, this value may determine which banks are presented to the customer.

I'm a little confused by this docstring. Is it saying that for LPMs that support multiple countries, this value represents the one that was actually shown to the customer?

Reply:
Both the name of the property and the docstring match what was implemented in the JS SDK. Since there's a property called `paymentType` (declared just above this one), calling it `paymentTypeCountryCode` makes it clear that it's the country code for the specific local payment type (as opposed to the merchant or customer's country, I suppose).

I interpret the docstring as saying that this property may determine which banks _will_ be presented to the customer. (Not which ones already have been.)

Reply:
Yeh, I just saw that we used the same phrasing as in JS. 

The plural on `banks` is throwing me off, since I assume this value can only represent 1 countryCode.

Reply:
I take it to mean that there could be more than one bank available in a given country.

Reply:
AH - yes. Thank you